packages/prisma/src/index.ts

```diff
@@ -140,20 +140,20 @@ export class DB {
     },
   }
 
-  static async mockup(name?: string): Promise<MockupDB> {
-    const dbName = name || `.mockup/${v4()}.db`
-    const dbPath = path.join(path.resolve('.'), dbName)
-    const dirPath = path.join(dbPath, '../')
+  static async mockup(dbPath?: string): Promise<MockupDB> {
+    const fullDbPath = dbPath || path.join(process.cwd(), `.mockup/${v4()}.db`)
+    // const dbPath = path.join(path.resolve('.'), dbName)
+    const dirPath = path.join(fullDbPath, '../')
     fs.mkdirSync(dirPath, { recursive: true })
-    const predefined = `${path.join(path.resolve(__dirname), '../mockup.db')}`
-    await fs.promises.copyFile(predefined, dbPath)
+    const predefined = path.join(path.resolve(__dirname), '../mockup.db')
+    await fs.promises.copyFile(predefined, fullDbPath)
     const db = new DB({
       datasources: {
-        sqlite: { url: `file://${dbPath}` },
+        sqlite: { url: `file:${fullDbPath}` },
       },
     })
     const terminate = async () => {
-      fs.unlinkSync(dbPath)
+      fs.unlinkSync(fullDbPath)
```

Comment:
@wanseob I think this is used mostly for tests but it seems dangerous to include this in a returned function. e.g. terminate is accidentally invoked in a running node and the database is lost.

It might make sense to put this in the tests instead of here.

Reply:
You're right, it has a possibility of blowing up the user's assets by some accident. Please make changes if you have any nice refactoring ideas on this!

Reply:
Ok, I split `mockup` into two functions: `mockup` and `testMockup`. Now the tests use the test function and everything else uses `mockup` which returns only the db.

Reply:
sounds very nice!